pages/dashboard/tracing/[caseId]/[contactId]/monitor.tsx

```diff
@@ -29,8 +36,68 @@ export default function Monitor() {
             <h1 className="text-2xl font-semibold">Tambah Pemantauan</h1>
           </div>
         </div>
-      </ConfirmedCasesHeader>
-      <DashboardPageContent>Content</DashboardPageContent>
+      </DashboardPageHeader>
+      <DashboardPageContent>
+        <div className="space-y-4">
+          <SectionCard className="space-y-6">
+            <div className="space-y-4">
+              <SectionCardHeader title="INFORMASI UTAMA" />
+              <div className="grid grid-cols-1 lg:grid-cols-2 gap-2">
+                <CaseInformationItem label="NIK" value="3401230000000001" />
+                <CaseInformationItem label="Nama Pasien" value="Muhammad Nuh" />
+                <CaseInformationItem label="Tanggal Lahir" value="13 Juni 1995 (26 thn 1 bln)" />
+                <CaseInformationItem label="Jenis Kelamin" value="Laki-laki" />
+                <CaseInformationItem label="No Telp/HP" value="085847475454" />
+                <CaseInformationItem label="Alamat" value="Jl HR Rasuna Said Kav C17" />
+              </div>
+            </div>
+            <div className="space-y-6 pt-6">
+              <SectionCardHeader title="PEMANTAUAN" />
+              <div className="space-y-2">
+                <div className="font-light">Tanggal dilakukan kunjungan ulang</div>
+                <div className="text-2xl font-bold">Minggu, 08 Agustus 2021</div>
```

Comment:
```suggestion
                <span className="font-light">Tanggal dilakukan kunjungan ulang</span>
                <span className="text-2xl font-bold">Minggu, 08 Agustus 2021</span>
```

Please don't use `div` for text elements. There are also more elements like this, but I highlighted this one for simplicity.